build/release/script/dev_kyuubi_vote.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o pipefail
+set -e
+
+RELEASE_DIR="$(cd "$(dirname "$0")"/..; pwd)"
+
+######### Please modify the variables ##########
+# release version, e.g. 1.4.0-incubating
+release_version=
+# release candidate number, e.g. 2
+release_rc_no=
+# previous release candidate number, e.g. 1, could be empty if it is the first vote
+prev_release_rc_no=
+# staging repository number, check it under https://repository.apache.org/content/repositories
+repo_no=
+################################################
+
+if [[ -z $release_version ]]; then
+  echo "Please input release version"
+  exit 1
+fi
+if [[ -z $release_rc_no ]]; then
+  echo "Please input release rc number"
+  exit 1
+fi
+if [[ -z $repo_no ]]; then
+  echo "Please input staging repository number, check it under https://repository.apache.org/content/repositories "
+  exit 1
+fi
+release_rc_tag=${release_version}-${release_rc_no}
+git_commit_hash=$(git rev-list -n 1 $release_rc_tag)
+
+if [[ ! -z "$prev_release_rc_no" ]]; then
+  prev_release_rc_tag=${release_version}-${prev_release_rc_no}
+  change_from_pre_commit="
+The commit list since the ${prev_release_rc_no}:
+https://github.com/apache/incubator-kyuubi/compare/${prev_release_rc_tag}...${release_rc_tag}
+"
+fi
+
+RELEASE_TEMP_DIR=${RELEASE_DIR}/tmp
+mkdir -p ${RELEASE_TEMP_DIR}
+DEV_VOTE=${RELEASE_TEMP_DIR}/${release_rc_tag}_dev_vote.temp
+
+cat <<EOF >${DEV_VOTE}
```

Comment:
Does the space after `EOF` matter? If not, change to `cat <<EOF > ${DEV_VOTE}`.

BTW, I only saw `cat > ${DEV_VOTE} <<EOF` before, not sure which style is popular

Reply:
thanks, it looks better, addreesed